packages/runtime/lib/config.js

```diff
@@ -89,10 +89,20 @@ async function _transformConfig (configManager, args) {
     if (configManager._fixPaths && service.config) {
       service.config = pathResolve(service.path, service.config)
     }
+
+    if (service.config) {
+      const manager = new ConfigManager({ source: pathResolve(service.config) })
```

Comment:
Use this method instead: https://github.com/platformatic/platformatic/blob/8361e7944d837eed258ef8cdfc095aaf7986c8d2/packages/config/lib/store.js#L259.

It will also try to load the necessary stackable.